app/views/dashboards/show.html.erb

```diff
@@ -222,6 +222,9 @@
   </div>
 </nav>
 <h2>Officer Dashboard</h2>
+    <% elsif current_member&.position == 'Pending' %>

```

Comment:
Should this be in the officer dashboard or the member dashboard section?

Reply:
Right now it's in the main dashboard because due to the link blacklist, the root page is the only thing that pending members have access to. The header up there doesn't show for some reason it can likely be removed.